enrollment-server/src/main/resources/application.properties

```diff
@@ -118,6 +118,7 @@ enrollment-server.presence-check.iproov.serviceHostname=${IPROOV_SERVICE_HOSTNAM
 spring.security.oauth2.client.provider.app.token-uri=http://localhost:6060/oauth/token
 
 # ZenID configuration
+enrollment-server.document-verification.zenid.additionalDocSubmitValidationsEnabled=${ZENID_ADDITIONAL_DOC_SUBMIT_VALIDATIONS_ENABLED:false}
```

Comment:
I am not sure we need to configure this aspect of the application. If the validations work properly, we should probably just use them. We already have a lot of configuration properties.

Reply:
My aim was to pilot the feature slowly, just to be able to turn it off when we would need.
I will still add a TODO for now, we can reconsider removing the configuration later.